README.md

```diff
@@ -1,7 +1,6 @@
 <p align="center">
-  <a href="https://jcristharif.com/msgspec/">
-    <img src="https://raw.githubusercontent.com/jcrist/msgspec/main/docs/source/_static/msgspec-logo-light.svg" width="35%" alt="msgspec" />
-  </a>
+  <img src="docs/source/_static/msgspec-logo-dark.svg#gh-light-mode-only" alt="msgspec" width="35%" height="auto" />
+  <img src="docs/source/_static/msgspec-logo-light.svg#gh-dark-mode-only" alt="msgspec" width="35%" height="auto" />
```

Comment:
These render on places other than github (e.g. pypi), but only github supports `gh-light-mode-only`/`gh-dark-mode-only`. We could hack around this by modifying the readme as part of the build or PyPI, but that seems like a lot of work. I'd rather not make this change for now.

Reply:
that sounds good, i wasnt sure of the scale it was used but thought there was a good chance it would be elsewhere